### ISSUE TITLE
Failed to activate the zentabs package

### ISSUE THREAD

**Issue by capaj** [ArnaudRinquin/atom-zentabs#38]
[Enter steps to reproduce below:]
1. Installed extensions with 25+ tabs opened, one of the tabs had unsaved file
2. upon being asked whether I want to save the file, I chose yes

**Atom Version**: 1.8.0
**System**: Ubuntu 16.04
**Thrown From**: [zentabs](https://github.com/ArnaudRinquin/atom-zentabs) package, v0.8.7
### Stack Trace

Failed to activate the zentabs package

```
At Cannot read property 'getGrammar' of undefined

TypeError: Cannot read property 'getGrammar' of undefined
    at TidyMarkdown.run (/home/capaj/.atom/packages/tidy-markdown/lib/index.coffee:37:14)
    at /home/capaj/.atom/packages/tidy-markdown/lib/index.coffee:21:12
    at TextBuffer.module.exports.TextBuffer.transact (/usr/share/atom/resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:828:18)
    at /home/capaj/.atom/packages/tidy-markdown/lib/index.coffee:19:14
    at Function.module.exports.Emitter.simpleDispatch (/usr/share/atom/resources/app.asar/node_modules/event-kit/lib/emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (/usr/share/atom/resources/app.asar/node_modules/event-kit/lib/emitter.js:125:28)
    at TextBuffer.module.exports.TextBuffer.saveAs (/usr/share/atom/resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:1091:20)
    at TextBuffer.module.exports.TextBuffer.save (/usr/share/atom/resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:1083:19)
    at TextEditor.module.exports.TextEditor.save (/usr/share/atom/resources/app.asar/src/text-editor.js:660:26)
    at Pane.module.exports.Pane.saveItem (/usr/share/atom/resources/app.asar/src/pane.js:718:18)
    at Pane.module.exports.Pane.promptToSaveItem (/usr/share/atom/resources/app.asar/src/pane.js:690:23)
    at Pane.module.exports.Pane.destroyItem (/usr/share/atom/resources/app.asar/src/pane.js:636:18)
    at /home/capaj/.atom/packages/zentabs/lib/zentabs-controller.coffee:80:17
    at Array.forEach (native)
    at ZentabsController.module.exports.ZentabsController.closeOverflowingTabs (/home/capaj/.atom/packages/zentabs/lib/zentabs-controller.coffee:66:14)
    at ZentabsController.module.exports.ZentabsController.initialize (/home/capaj/.atom/packages/zentabs/lib/zentabs-controller.coffee:42:6)
    at ZentabsController.View [as constructor] (/usr/share/atom/resources/app.asar/node_modules/atom-space-pen-views/node_modules/space-pen/lib/space-pen.js:184:25)
    at new ZentabsController (/home/capaj/.atom/packages/zentabs/lib/zentabs-controller.coffee:8:3)
    at /home/capaj/.atom/packages/zentabs/lib/zentabs.coffee:9:30
    at PaneContainer.module.exports.PaneContainer.observePanes (/usr/share/atom/resources/app.asar/src/pane-container.js:92:9)
    at Workspace.module.exports.Workspace.observePanes (/usr/share/atom/resources/app.asar/src/workspace.js:349:33)
    at Object.module.exports.activate (/home/capaj/.atom/packages/zentabs/lib/zentabs.coffee:8:39)
    at Package.module.exports.Package.activateNow (/usr/share/atom/resources/app.asar/src/package.js:183:20)
    at /usr/share/atom/resources/app.asar/src/package.js:156:32
    at Package.module.exports.Package.measure (/usr/share/atom/resources/app.asar/src/package.js:92:15)
    at /usr/share/atom/resources/app.asar/src/package.js:149:26
    at Package.module.exports.Package.activate (/usr/share/atom/resources/app.asar/src/package.js:146:34)
    at PackageManager.module.exports.PackageManager.activatePackage (/usr/share/atom/resources/app.asar/src/package-manager.js:538:21)
    at /usr/share/atom/resources/app.asar/node_modules/settings-view/lib/package-manager.js:452:29
    at exit (/usr/share/atom/resources/app.asar/node_modules/settings-view/lib/package-manager.js:73:16)
    at triggerExitCallback (/usr/share/atom/resources/app.asar/src/buffered-process.js:215:47)
    at /usr/share/atom/resources/app.asar/src/buffered-process.js:229:18
    at Socket.<anonymous> (/usr/share/atom/resources/app.asar/src/buffered-process.js:100:18)
    at emitOne (events.js:82:20)
    at Socket.emit (events.js:169:7)
    at Pipe._onclose (net.js:469:12)
```
### Commands

```
     -5:38.3.0 core:move-up (atom-text-editor.editor.is-focused)
     -5:38.2.0 snippets:next-tab-stop (atom-text-editor.editor.is-focused)
     -5:38.2.0 snippets:expand (atom-text-editor.editor.is-focused)
     -5:38.2.0 editor:indent (atom-text-editor.editor.is-focused)
     -5:37.7.0 core:paste (atom-text-editor.editor.is-focused)
  2x -5:36.8.0 core:backspace (atom-text-editor.editor.is-focused)
 12x -5:27.1.0 core:delete (atom-text-editor.editor.is-focused)
  2x -5:22.8.0 core:backspace (atom-text-editor.editor.is-focused)
  3x -5:07.4.0 tree-view:reveal-active-file (atom-workspace.workspace.scrollbars-visible-always.seti-ui-no-icons.theme-one-dark-syntax.theme-one-dark-ui)
     -3:05.9.0 tree-view:add-file (span.name.icon.icon-file-directory)
  6x -3:03.2.0 core:backspace (atom-text-editor.editor.mini.is-focused)
     -0:45.1.0 core:confirm (atom-text-editor.editor.mini.is-focused)
  2x -0:44.8.0 tree-view:reveal-active-file (atom-workspace.workspace.scrollbars-visible-always.seti-ui-no-icons.theme-one-dark-syntax.theme-one-dark-ui)
     -0:31.5.0 core:confirm (atom-text-editor.editor.mini.is-focused)
  2x -0:18.1.0 core:backspace (atom-text-editor.editor.mini.is-focused)
     -0:17.5.0 core:confirm (atom-text-editor.editor.mini.is-focused)
```
### Config

``` json
{
  "core": {}
}
```
### Installed Packages

``` coffee
# User
api-blueprint-preview, v0.7.0
atom-ternjs, v0.14.2
autocomplete-modules, v1.5.2
clipboard-history, v0.6.6
double-tag, v0.2.0
emmet, v2.4.3
es-identifier-highlight, v0.2.11
file-icons, v1.7.17
hyperclick, v0.0.37
js-hyperclick, v1.4.2
js-refactor, v0.7.3
language-SCSS, v0.4.0
language-api-blueprint, v1.0.2
language-gitignore, v0.2.0
line-diff-details, v1.3.5
linter, v1.11.14
linter-eslint, v7.2.4
merge-conflicts, v1.4.4
open-recent, v5.0.0
pigments, v0.31.1
react, v0.15.0
refactor, v0.9.2
synced-sidebar, v0.4.3
tidy-markdown, v3.0.1
zentabs, v0.8.7

# Dev
No dev packages
```

**Comment by ArnaudRinquin** [comment]
I noticed the same issue coupe of days ago. I'll try to have a look soonish.

**Comment by hron** [comment]
@ArnaudRinquin I'm sorry. I wanter to fix #35 instead of this.
